README.md

```diff
@@ -55,7 +55,7 @@ Start an anger instance against a given target.
     * `connections`: The number of connections to maintain to server. Each is an individual nes client.
     * `senders`: The number of connections that should be senders.
     * `responses`: The number of overall expected responses to be recieved during this run.
-    * `auth`: An `Object` passed to the nes client for authentication. [reference.][nes-auth]
+    * `auth`: A `Function` passed to the nes client for authentication. This is passed the `client` and `index` of that client as params. Must return the auth object options. [reference.][nes-auth]
```

Comment:
You should be able to know that if the `index` passed in is < `senders` then it's a sender _right now_. _buuuut_ what we can do is add a property to the `client` called `sender` which is `true` when a sender, or falsey (undefined) if not one. I can force it to be explicitly `false`, if you would prefer though.